src/components/PageSection.tsx

```diff
@@ -5,15 +5,16 @@ type Props = {
   description: string
   position: string
```

Comment:
can help update this from `position: string` to `position: 'right' | 'left'`

Reply:
https://github.com/rhdevs/rhdevs-website/pull/31/commits/9cf69f0b9bebddb256afaddc4e44ab28bcd04f4a

fixed